hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/om/GetServiceRolesSubcommand.java

```diff
@@ -94,7 +93,7 @@ public Void call() throws Exception {
       } else {
         printOmServerRoles(ozoneManagerClient.getServiceList());
       }
-    } catch (OzoneClientException ex) {
+    } catch (IOException ex) {
       System.out.printf("Error: %s", ex.getMessage());
```

Comment:
We can even completely remove `catch`.

Exception is handled by parent command (all other subcommands except `GetServiceRolesSubcommand` and `UpdateRangerSubcommand` rely on that).

Reply:
Thanks for catching this, let me fix both `GetServiceRolesSubcommand` and `UpdateRangerSubcommand`.